bookList.js

```diff
@@ -0,0 +1,27 @@
+class Booklist {// eslint-disable-line no-unused-vars
+  constructor() {
+    this.books = JSON.parse(localStorage.getItem('books')) || [];
+  }
+
+  // function to add books
+  addBook(title, author, id) {
+    this.books.push({ title, author, id });
```

Comment:
- [ ] OPTIONAL
It will be a good practice to favor the use of array methods such as `concat` over `push`, which doesn't mutate the array.